feeds/models.py

```diff
@@ -1,36 +1,57 @@
 from __future__ import unicode_literals
 
 from django.db import models
-import uuid
+from feeds.mixins import UUIDMixin
+# from django.contrib.postgres.fields import JSONField
 
 # Create your models here.
 
 
-class AuthToken(models.Model):
-    uuid = models.CharField(max_length=64, default=uuid.uuid4,
-                            db_index=True)
+class AuthToken(UUIDMixin):
     screen_name = models.CharField(max_length=60, unique=True)
     # Should they be stored directly
     access_token = models.CharField(max_length=120)
     access_token_secret = models.CharField(max_length=120)
 
 
-class TwitterAccount(models.Model):
+class TwitterAccount(UUIDMixin):
     screen_name = models.CharField(max_length=60, unique=True)
     followed_from = models.ForeignKey(AuthToken, on_delete=models.CASCADE)
```

Comment:
Why is this a `ForeignKey` and not a `ManytoMany`? Also, `last_updated` is a little tricky, if this is a foreign key, unless each Account is checked once for each `followed_from` account.